layers/+tools/shell/README.org

```diff
@@ -34,23 +40,51 @@ To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =shell= to the existing =dotspacemacs-configuration-layers= list in this
 file.
 
+** Install vterm
+=vterm= is the latest addition to Emacs' set of terminal emulators and the only
+one to be implemented in C, leveraging =libvterm=. It is the only one in Emacs at
+the moment to be as fast as a standalone terminal and to support =ncurses=, =vim=,
+=top=, =htop= and the likes.
```

Comment:
Add list of dependencies (as of today), plus link to emacs-libvterm dependency documentation. Include the --with-modules requirement and how to check if Emacs was compiled with --with-modules (variable `system-configuration-options` contains "--with-modules" or variable `system-configuration-features` contains "MODULES").

Reply:
Try to explain what emacs-libvterm tries to do when it insalls itself, which happens the first time the user tries to open vterm (compile emacs-libvterm.c, download stuff? etc.).

Reply:
Done ✨